doc/changes/latest.inc

```diff
@@ -68,6 +68,7 @@ Bugs
 - Fix bug in :func:`mne.preprocessing.compute_maxwell_basis` where using ``int_order=0`` would raise an error (:gh:`11562` by `Eric Larson`_)
 - In the legacy function :func:`mne.viz.plot_epochs_psd_topomap`, the parameter ``names`` now works again (:gh:`11563` by `Daniel McCloy`_)
 - Fix :func:`mne.io.read_raw` for file names containing multiple dots (:gh:`11521` by `Clemens Brunner`_)
+- Fix bug in :meth:`mne.Annotations.rename` where replacements were not done correctly (:gh:`11666` by `Eric Larson`_)
```

Comment:
@Qwinpin I'd like to give you the bulk of the credit here. If you let me know your preferred name + URL + email that you use for GitHub commits (for co-author credit) I'm happy to add you here!

Reply:
Hi, you can credit me like Timur Sokhin, qwinpin@gmail.com. I'm glad I could help!